e2e/pages/app/badge/badge.testpage.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="badge-container">
```

Comment:
Although it wasn't noted in the ticket, I think the plan was to add a large number of static examples representing all the mockup examples that Roland provided, and just take a screenshot of that.

Looking at the mockups, you could just set up arrays of each testable property and loop through them in the template, e.g.
```ts
sizes = ['small', 'medium', 'large'];
```
```html
<button *ngFor="let size of sizes" ... [uxBadge]="99" [size]="size">...</button>
```

You could save horizontal space by using icon buttons instead of text buttons.

It would not be necessary to test all combinations, but I would like to see coverage of most of what Roland has mocked up.